Swiftfin/Views/VideoPlayer/Overlays/ChapterOverlay.swift

```diff
@@ -65,8 +65,8 @@ extension VideoPlayer.Overlay {
                             .foregroundColor(accentColor)
                     }
                 }
-                .padding(.leading, safeAreaInsets.leading)
-                .padding(.trailing, safeAreaInsets.trailing)
+                .padding(.leading, UIDevice.isIPad ? nil : safeAreaInsets.leading)
```

Comment:
I would prefer the way I have done this in `PillHStack/PosterHStack`, for example:

```swift
MyView()
	.padding(.leading) // padding on all platforms
	.if(UIDevice.isIPad) { view in
		view.padding(.leading)
	}
```

Additionally, I would like the headers and the buttons to align, so this extra padding should be put on the chapter `HStack` as well.

Reply:
Ah sorry, I didn't read your comment closely enough. To be sure, this is what we want right? Lmk if not

| Before | After |
|--------|--------|
| ![Simulator Screenshot - iPad Air 4th - 2023-09-24 at 09 17 00](https://github.com/jellyfin/Swiftfin/assets/22553678/69b79679-e090-4c9e-b433-30932b196617) | ![image](https://github.com/jellyfin/Swiftfin/assets/22553678/01594442-af92-47b8-84ff-190ce342db65) |

Reply:
Yep, just like that 👍